mixins/interactive/test/interactive-mixin.test.js

```diff
@@ -57,8 +56,8 @@ describe('InteractiveMixin', () => {
 			toggle.click();
 			await new Promise(resolve => setTimeout(resolve, 0));
 			expect(getComposedActiveElement()).to.equal(elem.shadowRoot.querySelector('.content-button'));
-			keyDown(toggle, 27);
-			await new Promise(resolve => setTimeout(resolve, 0));
+			sendKeysElem(toggle, 'press', 'Escape');
+			await waitUntil(() => getComposedActiveElement().classList.contains('interactive-toggle'));
```

Comment:
`sendKeysElem` seems to be a lot slower, so waiting for something more concrete.

Reply:
Looks like yes 🤦. Totally forgot this function was `async` 😅, every other place using it needs to wait for a certain event anyways

Reply:
Yeah it's a little weird to remember at first. There will definitely be uses of both, like:

```javascript
sendKeysElem(elem, 'press', 'Tab');
await oneEvent(elem, 'blur');
```
and:
```javascript
await sendKeysElem(elem, 'press', 'Tab');
expect(elem.classList).to.not.include(['active']);
```

Reply:
Yeah exactly. It's so nice though, so far everything "just works" (as long as you remember the above 😝)